src/lib/opal_parser.c

```diff
@@ -113,6 +113,26 @@ static int append_medium_atom_bytes_header(uint8_t *buf, size_t len, int data_le
 	return 0;
 }
 
+static int append_long_atom_bytes_header(uint8_t *buf, size_t len, int data_len)
+{
```

Comment:
please fix coding style to proper indentation

Reply:
will fix it